tecken/download/views.py

```diff
@@ -325,11 +243,7 @@ def missing_symbols_csv(request):
 
     # By default, only do those updated in the last 24h
     qs = MissingSymbol.objects.filter(
-        modified_at__gte=date,
-        # This is a trick to immediately limit the symbols that could
-        # be gotten from a Microsoft download.
-        symbol__iendswith=".pdb",
-        filename__iendswith=".sym",
```

Comment:
Now that I'm re-reading this, I think we need to re-add it. It's not related to the `download_from_microsoft` feature.